tensorflow_addons/layers/polynomial.py

```diff
@@ -40,15 +40,12 @@ class PolynomialCrossing(tf.keras.layers.Layer):
 
     Example:
 
-    ```python
-    # after embedding layer in a functional model:
-    input = tf.keras.Input(shape=(None,), name='index', dtype=tf.int64)
-    x0 = tf.keras.layers.Embedding(input_dim=32, output_dim=6))
-    x1 = PolynomialCrossing(projection_dim=None)((x0, x0))
-    x2 = PolynomialCrossing(projection_dim=None)((x0, x1))
-    logits = tf.keras.layers.Dense(units=10)(x2)
-    model = tf.keras.Model(input, logits)
-    ```
+    >>> input = np.random.randint(10,size=(10,5))
+    >>> x0 = tf.keras.layers.Embedding(input_dim=10, output_dim=3)(input)
+    >>> x1 = PolynomialCrossing(projection_dim=None)((x0, x0))
+    >>> x2 = PolynomialCrossing(projection_dim=None)((x0, x1))
+    >>> logits = tf.keras.layers.Dense(units=10)(x2)
+    >>> model = tf.keras.Model( logits)
```

Comment:
Could you display the model output here?

Reply:
@Harsh188. if you are asking for something like `print(model)` or just `model`  then the answer is no because every time the code runs model object has a different address. 
<tensorflow.python.keras.engine.training.Model at 0x7f4c64f21d68>